docs/docs/build/bridged-token-addresses/index.md

```diff
@@ -0,0 +1,32 @@
+# Bridged Token Addresses
+
+Various ERC-20 tokens originally deployed to Ethereum also have corresponding "bridged" representations on BOB Mainnet.
+
+:::info Listed Tokens
```

Comment:
Maybe a more descriptive heading that summarizes this info box, such as "Do Your Own Research" or "Not An Endorsement" instead of "Listed Tokens"